app/forms/publish/course_study_site_form.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Publish
+  class CourseStudySiteForm < BaseCourseForm
+    FIELDS = %i[study_site_ids].freeze
+
+    attr_accessor(*FIELDS)
+
+    validate :no_study_sites_selected
```

Comment:
Can't we use `validates :study_site_ids, presence: true`?

Reply:
Yes, done here https://github.com/DFE-Digital/publish-teacher-training/pull/3705/commits/ec8986de6b21f8947653fcd83062cce30902a15b